src/pages/SearchView.jsx

```diff
@@ -15,9 +15,15 @@ import "rc-pagination/assets/index.css";
 
 export function sendSearchFromForm(event, history) {
   const searchKeyword = event.target.elements.searchKeyword.value;
-  if (searchKeyword) {
-    history.push(getSearchURL({ q: searchKeyword }));
-  }
+
+  let finalSearchKeyword = createSearchQuery({ year_start: 1892, year_end:1904, query:searchKeyword });
```

Comment:
Could you put 1892 and 1904 in a constants file (https://github.com/TheStanfordDaily/archives-web/blob/master/src/helpers/constants.js) so that we aren't hardcoding it here?